src/WCSimDetectorMessenger.cc

```diff
@@ -922,7 +922,7 @@ void WCSimDetectorMessenger::SetNewValue(G4UIcommand* command,G4String newValue)
 
     if(command == WCConstruct) {
       //If the OD geometry has been changed, then reconstruct the whole tank with the proper recalculated dimensions
-      if (WCSimDetector->GetODEdited() == true) {WCSimDetector->UpdateODGeo();}
+      //if (WCSimDetector->GetODEdited() == true) {WCSimDetector->UpdateODGeo();}
```

Comment:
this is problem if the OD geometry is not updated

Reply:
Funny, I don't see this issue on my setup. Maybe there is a deeper reason, but it's better not to commit this change which affects the simulation output

Reply:
should be resolved in [PR403](https://github.com/WCSim/WCSim/pull/403)